frontend/src/metabase/lib/formatting.js

```diff
@@ -54,7 +54,7 @@ import type {
 import type { ClickObject } from "metabase-types/types/Visualization";
 
 // a one or two character string specifying the decimal and grouping separator characters
-export type NumberSeparators = ".," | ", " | ",." | ".";
+export type NumberSeparators = ".," | ", " | ",." | "." | "’.";
```

Comment:
@etoulas I think the `'` and `.` backwards here

Reply:
You are right! I wonder where this is used, because I couldn't spot the difference.

Reply:
They types are only used in our flow type checking. They don't make it into the compiled app code. It looks like the flow check passed, so I guess that type isn't used effectively enough. Still good to keep it up to date though.